### PR TITLE
feat(#261): add new folder with deployment commands and delete deploy…

### PR DESCRIPTION
**Pull request by juandiegocv27**
This is the issue #261.

Just modified and add some features for Slot CLI documentation that will bring great context with:

-deleted deployments page.
-added create, delete, update, describe, list, logs and help command pages.

image

@kariy.

### CODE REVIEW

**Comment by juandiegocv27** [comment]
Hi, @kariy I have an error, in this part, could you check it, I'll fix it as soon as possible

**Review by kariy — COMMENTED** [review]
hey @juandiegocv27 thanks for the work!

can you also update the sidebar [config](https://github.com/dojoengine/book/blob/76bc33372761cd40d4121b7facd6c1373b3d93f0/vocs.config.ts#L188) so that the new pages would be accessible from the sidebar ?

Screenshot 2024-05-13 at 9 12 24 AM

**Comment by juandiegocv27** [comment]
> hey @juandiegocv27 thanks for the work!
> 
> 
> 
> can you also update the sidebar [config](https://github.com/dojoengine/book/blob/76bc33372761cd40d4121b7facd6c1373b3d93f0/vocs.config.ts#L188) so that the new pages would be accessible from the sidebar ?
> 
> 
> 
> Screenshot 2024-05-13 at 9 12 24 AM
> 
> 

Ok sure 🫡

**Comment by juandiegocv27** [comment]
Hi @kariy, I just fix it, and agreggate some features, hope for feedback

**Comment by juandiegocv27** [comment]
HI @kariy @milancermak @milancermak, is this PR is still relevant? Please let me know if there's anything else you’d like me to add or if you have any feedback. I'm happy to make any changes you need. Thanks

**Comment by ponderingdemocritus** [comment]
@juandiegocv27 could you fix the conflict

**Comment by juandiegocv27** [comment]
@ponderingdemocritus sure, but, talking in discord with @kariy told me to erase the file deployments.md, so I could return it, but all that I do was put more stuff and make deployments a folder

**Comment by juandiegocv27** [comment]
@ponderingdemocritus @kariy just fix the conflict

**Comment by juandiegocv27** [comment]
@ponderingdemocritus @kariy just fix the conflict, but new conflict appear, now the conflict dont tell me what to fix

**Comment by juandiegocv27** [comment]
@ponderingdemocritus @kariy @milancermak @ptisserand, just resolved conflicts

**Review by kariy — CHANGES_REQUESTED** [review]

**Comment by juandiegocv27** [comment]
@ponderingdemocritus @kariy @milancermak @ptisserand Fix all the code and ready to merge: 
image

waiting for more feedback 😎🚀

File: docs/pages/toolchain/slot/Deployments/delete.md
Comment (juandiegocv27) [reply docs/pages/toolchain/slot/Deployments/delete.md]: this is ready

File: docs/pages/toolchain/slot/Deployments/describe.md
Comment (juandiegocv27) [reply docs/pages/toolchain/slot/Deployments/describe.md]: already

File: docs/pages/toolchain/slot/Deployments/help.md
Comment (juandiegocv27) [reply docs/pages/toolchain/slot/Deployments/help.md:1]: ok

File: docs/pages/toolchain/slot/Deployments/delete.md
Comment (juandiegocv27) [reply docs/pages/toolchain/slot/Deployments/delete.md]: I just put it some kind like that, tell me what you about it

**Review by glihm — CHANGES_REQUESTED** [review]
Thanks for the work here @juandiegocv27!

Some important points:
1. consistency: `slot` should be formatted with consistency as sometimes you use `Slot` and sometimes Slot.
2. formatting: I've left some commands on formatting, not on all the lines that require changes, you may do a general review of formatting.

**Comment by juandiegocv27** [comment]
> Thanks for the work here @juandiegocv27!
> 
> 
> 
> Some important points:
> 
> 1. consistency: `slot` should be formatted with consistency as sometimes you use `Slot` and sometimes Slot.
> 
> 2. formatting: I've left some commands on formatting, not on all the lines that require changes, you may do a general review of formatting.

@glihm thanks a lot for the feedback, I'll work on it as fast as possible ⚡️

File: docs/pages/toolchain/slot/index.md
Comment (juandiegocv27) [reply docs/pages/toolchain/slot/index.md:3]: thanks for the feedback, already making changes

**Comment by juandiegocv27** [comment]
hI @glihm, @kariy, fix somethings in this issue, ready to merge🚀

File: docs/pages/toolchain/slot/deployment-commands/create.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/create.md:16]: We can remove the '' if the project has not spaces in it.
Same for the version. We can also update for a more recent version that exists:
```
slot d create example_project katana --version 1.0.0-alpha.1
```

File: docs/pages/toolchain/slot/deployment-commands/create.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/create.md:22]: Same here to be consistent, without the ''.

File: vocs.config.ts
Comment (glihm) [inline vocs.config.ts:271]:
```suggestion
		              text: "Deployments",
```

File: docs/pages/toolchain/slot/deployment-commands/delete.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/delete.md:4]:
```suggestion

```

File: docs/pages/toolchain/slot/deployment-commands/delete.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/delete.md:16]: Same here let's show and example with snake case without `''`.

File: docs/pages/toolchain/slot/deployment-commands/update.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/update.md:16]: Same comment for `''` and version. 👍

File: docs/pages/toolchain/slot/deployment-commands/describe.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/describe.md:17]: Same comment for syntax, and we may actually add an output to show what's expected.

File: docs/pages/toolchain/slot/deployment-commands/logs.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/logs.md:27]: Those sections with response and output are great. Could we generalize it to the other commands showing real output + comments on it when required?

File: docs/pages/toolchain/slot/deployment-commands/logs.md
Comment (glihm) [inline docs/pages/toolchain/slot/deployment-commands/logs.md:5]: Comment that applies for all, let's use `##` here as the first level if `#`.

Like so, if we want to insert a new subsection, we can use `###`.

**Review by glihm — COMMENTED** [review]
@juandiegocv27 thanks for the work here! Some comments that also apply to all commands.

Once done, this should be ready for merging!
Thank you for the iterations, let's know if you have any question!

**Comment by juandiegocv27** [comment]
> @juandiegocv27 thanks for the work here! Some comments that also apply to all commands.
> 
> Once done, this should be ready for merging! Thank you for the iterations, let's know if you have any question!

Thank you for the feedback @glihm, I´ll make the changes as fast as possible

**Comment by glihm** [comment]
> > @juandiegocv27 thanks for the work here! Some comments that also apply to all commands.
> > Once done, this should be ready for merging! Thank you for the iterations, let's know if you have any question!
> 
> Thank you for the feedback @glihm, I´ll make the changes as fast as possible

No rush, thank you for your contribution. 👍

**Comment by juandiegocv27** [comment]
> > > @juandiegocv27 thanks for the work here! Some comments that also apply to all commands.
> > > Once done, this should be ready for merging! Thank you for the iterations, let's know if you have any question!
> > 
> > 
> > Thank you for the feedback @glihm, I´ll make the changes as fast as possible
> 
> No rush, thank you for your contribution. 👍

Already fix and put response and output in every command😎 @glihm

**Review by kariy — APPROVED** [review]
lgtm

**Comment by juandiegocv27** [comment]
> lgtm

thank you so much

**Pull request merged**
